src/lib.rs

```diff
@@ -22,8 +22,8 @@ pub struct SlotConfig {
 #[allow(missing_copy_implementations)]
 #[derive(Clone)]
 pub struct InitialBudget {
-    pub mem: u64,
```

Comment:
@satran004 I can isolate this in smaller PR or we leave it for now?

Reply:
@matiwinnetou Yes. I think we can keep this change in a separate PR.